src/Pages/falcon9/Falcon9.js

```diff
@@ -3,19 +3,76 @@ import Featuer from "./feature/Featuer";
 import Stats from "./stats/Stats";
 import Video from "./Video/Video";
 import Engines from "./Engines/Engines";
+import Footer from "../../Components/footer/Footer";
 import Gallery from "./Gallery/Gallery";
-
+import delayUnmounting from "./YoutubeModal/DelayUnmounting";
+import Modal from "./YoutubeModal/YoutubeModal";
 import "./Falcon9.scss";
 
 class Falcon9 extends Component {
+  state = {
+    orgBodyClassName: "",
+    nowScrollPosition: 0,
+    isModalRendered: false,
+    modalHaventRenderBefore: true,
+  };
+
+  componentDidMount() {
+    const body = document.getElementsByTagName("body")[0];
+    const orgBodyClassName = body.className;
+    this.setState({ orgBodyClassName: orgBodyClassName });
```

Comment:
```suggestion
    this.setState({ orgBodyClassName });
```
객체 단축 속성명을 사용하면 하나만 써줘도 되겠네요!